packages/contracts/contracts/optimistic-ethereum/OVM/bridge/messaging/OVM_L1CrossDomainMessenger.sol

```diff
@@ -139,7 +211,15 @@ contract OVM_L1CrossDomainMessenger is iOVM_L1CrossDomainMessenger, Abs_BaseCros
 
     /**
      * Replays a cross domain message to the target messenger.
-     * @inheritdoc iOVM_L1CrossDomainMessenger
+     * In case an L1 to L2 message is not successfully executed in the L2 messenger,
+     * this function can be used to replay the message.
+     * There is no limit to how many time this can be called,
+     * but replay protection is handled by the L2 messenger.
```

Comment:
```suggestion
     * There is no limit to how many times this can be called,
     * so replay protection is handled within L2 by the OVM_L2CrossDomainMessenger.
```